js/checklist.js

```diff
@@ -111,13 +113,15 @@ CheckListWidget.prototype.handleClearChecksEvent = function(event) {
 
 // When the user starts typing, change the pencil icon into a checkbox
 CheckListWidget.prototype.handleNewItemTypingEvent = function(event) {
```

Comment:
Why don't we just remove this whole block then instead of leaving in commented-out code